src/RuleSet/RuleSet.php

```diff
@@ -222,34 +226,34 @@ public function getRulesAssoc($mRule = null)
      * Note: this is different from pre-v.2.0 behaviour of PHP-CSS-Parser, where passing a Rule instance would
      * remove all rules with the same name. To get the old behaviour, use `removeRule($rule->getRule())`.
      *
-     * @param Rule|string|null $mRule
-     *        pattern to remove. If $mRule is null, all rules are removed. If the pattern ends in a dash,
+     * @param Rule|string|null $searchPattern
+     *        pattern to remove. If this is `null`, all rules are removed. If the pattern ends in a dash,
      *        all rules starting with the pattern are removed as well as one matching the pattern with the dash
      *        excluded. Passing a Rule behaves matches by identity.
      */
-    public function removeRule($mRule): void
+    public function removeRule($searchPattern): void
     {
-        if ($mRule instanceof Rule) {
-            $sRule = $mRule->getRule();
-            if (!isset($this->aRules[$sRule])) {
+        if ($searchPattern instanceof Rule) {
+            $propertyToRemove = $searchPattern->getRule();
+            if (!isset($this->rules[$propertyToRemove])) {
                 return;
             }
-            foreach ($this->aRules[$sRule] as $key => $rule) {
-                if ($rule === $mRule) {
-                    unset($this->aRules[$sRule][$key]);
+            foreach ($this->rules[$propertyToRemove] as $key => $rule) {
+                if ($rule === $searchPattern) {
+                    unset($this->rules[$propertyToRemove][$key]);
                 }
             }
         } else {
-            foreach ($this->aRules as $sName => $aRules) {
+            foreach ($this->rules as $ruleName => $rules) {
```

Comment:
`$propertyName` rather than `$ruleName`

Reply:
Actaully I'm not sure.

Reply:
If it's not a nested `RuleSet` then 'property name' is correct.  But I'm not sure otherwise.  Perhaps `$propertyOrRuleName` may be better throughout.